### DISCUSSION TITLE
第一次上绳网，有绳匠带我去空洞整点以太么？

### DISCUSSION THREAD

**Discussion by NEET10086** [share121/inter-knot#2553] [其他]
![image](https://github.com/user-attachments/assets/eca68bd0-607a-4c7c-8164-4aabb0ea6028)
RT，不要菜鸟绳匠，菜鸟绳匠，菜鸟绳匠，重要的事情说三遍！

**Comment by hua-yuanshuai** [comment]
我可以给你推荐专业的绳匠，但你要把你的以太适应检测发出来。免得刚进空洞就以骇化了<img src="https://img-static.mihoyo.com/communityweb/upload/1a25d24ab81aaa88543ffad7cbf3f944.png" alt="绝区零_暗中观察" width="75" >

**Comment by Pheathon** [comment]
这个委托我接了

**Comment by sdfefoewfwfw** [comment]
进空洞前买装备认准桑歌马哈巴依，保证有稀奇货！！！

**Reply by sdfefoewfwfw** [reply]
![R$ZGH)8{64M6`(7~Z6B0E2Q](https://github.com/user-attachments/assets/51dac8d6-5dc1-4d51-bd91-1abd4d596314)

**Reply by lixiao189** [reply]
虚空终端还能上绳网么

**Reply by AnonCN** [reply]
赤王科技，兄弟

**Comment by GrunderIndustries** [comment]
需要装备的话请认准创立者工业
便宜实惠性价比高

**Comment by Huanshenxuanlong** [comment]
看看最近有谁等级升的快，说不定是被封号的法厄同，蹭个车估计大佬也不会介意的
